### PR TITLE
Remove leading double slash in shutdown script

### PR DESCRIPTION
**Pull request by mahdi-torabi**

### CODE REVIEW

File: src/ec2/userdata.ts [hunk 1]
```diff
@@ -38,15 +38,17 @@ export class UserData {
       "#!/bin/bash",
       `shutdown -P +${this.config.ec2InstanceTtl}`,
       "CURRENT_PATH=$(pwd)",
-      `echo "./config.sh remove --token ${runnerRegistrationToken.token} || true" > $CURRENT_PATH/shutdown_script.sh`,
-      `echo "shutdown -P +1" > $CURRENT_PATH/shutdown_script.sh`,
-      "chmod +x $CURRENT_PATH/shutdown_script.sh",
-      `echo "./config.sh remove --token ${runnerRegistrationToken.token} || true" > $CURRENT_PATH/shutdown_now_script.sh`,
```
[the remark this answers is not in the document]
Comment (mahdi-torabi) [reply src/ec2/userdata.ts:44]: Ah shoot you are right. Gonna fix it

File: src/ec2/userdata.ts [hunk 2]
```diff
@@ -38,15 +38,17 @@ export class UserData {
...
-      `echo "shutdown -h now" > $CURRENT_PATH/shutdown_now_script.sh`,
-      "chmod +x $CURRENT_PATH/shutdown_now_script.sh",
-      "export ACTIONS_RUNNER_HOOK_JOB_COMPLETED=$CURRENT_PATH/shutdown_script.sh",
+      'CURRENT_PATH="${CURRENT_PATH%/}"',
+      "SHUTDOWN_SCRIPT_PATH=$CURRENT_PATH/shutdown_script.sh",
+      `echo "./config.sh remove --token ${runnerRegistrationToken.token} || true" > $SHUTDOWN_SCRIPT_PATH`,
+      `echo "shutdown -P +1" > $SHUTDOWN_SCRIPT_PATH`,
```
Comment (martin-ncs) [inline src/ec2/userdata.ts:44]: You are using `>` twice, which will overwrite the entire file, so the first `echo` was never executed. I think you want `>>` instead

Comment (martin-ncs) [reply src/ec2/userdata.ts]: even better, you can use...

```shell
cat > $SHUTDOWN_SCRIPT_PATH <<EOF
#!bash
./config.sh remove --token ${runnerRegistrationToken.token}
shutdown -P +1
EOF
```

Comment (mahdi-torabi) [reply src/ec2/userdata.ts]: I removed that variable and reverted the script. Just kept the part which removes the trailing slash. I wasn't comfortable making too many changes to this script.

Testing it is a lot of pain

**Pull request merged**